DOCS/man/options.rst

```diff
@@ -4708,6 +4708,18 @@ The following video options are currently all specific to ``--vo=gpu`` and
     functionality to receive a device, such as D3D11VA or DXVA2's DXGI
     mode, will be affected by this choice.
 
+``--d3d11-output-format=<auto|rgba8|bgra8|rgb10_a2|rgba16f>``
+    Select a specific D3D11 output format to utilize for D3D11 rendering.
```

Comment:
Should say that auto is the default, which may be obvious, but not necessarily.

Reply:
Added mention of "auto" being the default.